spy/tests/compiler/test_basic.py

```diff
@@ -354,6 +354,67 @@ def bar() -> None:
         mod.bar()
         assert mod.x == 3
 
+    def test_boolops_short_circuit(self):
+        mod = self.compile("""
+        var x: i32 = 0
+
+        def make_true() -> bool:
+            x = x + 1
+            return True
+
+        def make_false() -> bool:
+            x = x + 1
+            return False
```

Comment:
I think it's a good idea to be consistent with the names `counter` and `bump_{true,false}` which are used in the test below